starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <link rel="stylesheet" href="./css/main.css">
+  <meta charset="UTF-8">
   <title>Apple Pie Recipe</title>
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+<div class="header">
+  <h1>Apple Pie</h1>
+</div>
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+<section class="intro container">
+  <p>This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+  <img class="recipeInfo"  src="./images/recipe-info.png" alt="Recipe information" align="middle">
+</section>
 
-Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
-lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-that it does not run off.
+<hr></hr>
+
+<section class="ingredients container">
+  <h3>Ingredients</h3>
+  <ul>
+    <li>1 recipe pastry for a 9 inch double crust pie</li>
+    <li>1/2 cup unsalted butter</li>
+    <li>3 tablespoons all-purpose flour</li>
+    <li>1/4 cup water</li>
+    <li>3 tablespoons all-purpose flour</li>
+    <li>1/4 cup water</li>
+    <li>1/2 cup white sugar</li>
+    <li>1/2 cup packed brown sugar</li>
+    <li>8 Granny Smith apples - peeled, cored and sliced</li>
+  </ul>
+  <img class="cookingInfo" src="./images/cooking-info.png" alt="Cooking information">
+</section>
+
+<hr></hr>
+
+<section class="directions container">
+  <h3>Directions</h3>
+  <p>1) Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce temperature and let simmer.</p>
```

Comment:
I'm pretty sure there's a better and more generic way to do 1). Try using 
```
<ol>
   <li><li>
</ol>
```
and on the css side:
```
ol.bracket > li:before {
  content: counter(section) ") ";
}
li { 
   counter-increment: section;
}
```
Can you try to find some other solution too?